frontend/src/components/adminDetails/companyCard/CompanyCard.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import locationSvg from '../assets/Location.svg';
+import style from './company.module.css';
+import useAdminStore from '../../../store/admin.store';
+import { CompanyPropsType } from '../types/types';
+import { useEffect } from 'react';
+import Map from '../../../map/Map';
+
+const CompanyCard: React.FC<CompanyPropsType> = ({ company }) => {
+  const [isClicked, setIsClicked] = useState<boolean>(false);
+  const deleteCompany = useAdminStore((state) => state.deleteCompany);
+  const addcompany = useAdminStore((state) => state.addcompany);
+  const [isShowAbout, setIsShowAbout] = useState(false);
+
+  const handleDenyClick = async (id: number) => {
+    setIsClicked(true);
+    setTimeout(() => {
+      deleteCompany(id);
+    }, 300);
```

Comment:
Why do we need the timeout here?

Reply:
Hi , If I don't use it the fade away style will not work :pleading_face: 
I tried with many approach this the only way its working